src/components/AddedFeatures.js

```diff
@@ -19,4 +22,13 @@ const AddedFeatures = props => {
   );
 };
 
-export default AddedFeatures;
+
+function mapStateToProps(state) {
+  return {
+    add: state.add
```

Comment:
Try being a bit more descriptive with your variables. At a glance, I'm not sure what the add slice of state holds.